.circleci/config.yml

```diff
@@ -521,29 +516,31 @@ jobs:
           report_failure: << parameters.report_failure >>
           message: "create-xcframework"
 
-  # This job is responsible for building and uploading the xcframework bundle to S3.
-  # This job will also create a PR in api-downloads for SDK Registry access.
-  ios-build-release:
+  # This job:
+  # - builds XCFrameworks
+  # - uploads them to S3
+  # - creates SDK Registry PR
+  # - builds API docs
+  # - creates a draft release on GitHub
+  release:
     <<: *base-job
     steps:
-      - checkout
-      - add_ssh_keys:
-           fingerprints:
-             - "3b:cd:47:bf:57:9c:e5:36:b0:4d:5f:12:5e:d3:b3:3e"
-      - install-mbx-ci
-      - configure-environment
-      - ensure-text-exists-in-commit:
-          commit-text: "[release]"
-          always-run: << parameters.device-tests-always-run >>
       - run:
-          name: Extract Version From Commit
+          name: Verify that tag $CIRCLE_TAG adheres to the Semantic Versioning format
           command: |
-            GIT_COMMIT_MSG=`git log --format=oneline -n 1`
-            export VERSION=$(echo $GIT_COMMIT_MSG| cut -d' ' -f3)
-            echo "export VERSION=$VERSION" >> $BASH_ENV
```

Comment:
Can we discuss the drop of the `$VERSION` env variable? It looks like a good idea to share actual `VERSION` with all nested processes. We just have to parse `CIRCLE_TAG` for a semver instead of git commit message. 

We had the same practice in Search SDK and was cool:
https://github.com/mapbox/mapbox-search-ios/blob/32028ae77fee0fbe0dadb5fdde9d1d3a4a2db512/.circleci/config.yml#L347-L356

Reply:
Another reason I would prefer VERSION over CIRCLE_TAG is variable support in custom scripts. It's much more intuitive to pass `VERSION=10.1.0-rc2 ./run-me.sh` than passing a `CIRCLE_TAG` in the local environment (in case we have to call versioned scripts locally someday).

Reply:
I'm open to this. I'll follow up with you directly.